.gitignore

```diff
@@ -1,4 +1,7 @@
 .kamu
 .kamu.local
 .ipynb_checkpoints
-__pycache__
\ No newline at end of file
+__pycache__
+
+net.rocketpool/image/.odf_new_etag
+net.rocketpool/image/.odf_new_has_more
```

Comment:
It's better to add another `.gitignore` into `net.rocketpool/image/` for these files - this way if we remove/rename the image we don't risk the chance of forgetting to update the root `.gitignore`.

Reply:
a good point -- will do

Reply:
Done